src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp

```diff
@@ -751,9 +755,17 @@ void jit_store_emitter::store_bytes(const Vmm &vmm, const Xbyak::Reg64 &reg, int
 
         // 64/32/16/8 with one go
         // tail 7 bytes for lower or upper xmm
+        bool ext8bit = false;
         switch (bytes_to_store) {
             case 0: break;
-            case 1: h->uni_vpextrb(addr(start_bytes), xmm, 0); break;
+            case 1:
+                h->uni_vmovq(Reg64(aux_gpr_idxs[0]), xmm);
```

Comment:
Does it really bring any perf improvement comparing with `uni_vpextrb`? If yes it looks weird why other cases (bytes: 2, 3, etc) are not changed as well.
This question applicable to load emitter as well,

Reply:
Based on experiment, there are perf improvement on small number(1-4 byte). Extend to these cases. Thank you Dmitry.